blueocean-analytics-tools/src/main/resources/io/jenkins/blueocean/analyticstools/AnalyticsTools/header.jelly

```diff
@@ -1,32 +1,34 @@
-          ]]>
+                  return normalizedUrl;
+                }
+                var transformer = function(payload) {
+                  payload.data.request.user_ip = '0.0.0.0';
+                  payload.data.request.url = normalizeURL(window.location.pathname);
+                };
+                var _rollbarConfig = {
+                  accessToken: "81f3134dedf44871b9cc0a347b1313df",
+                  captureUncaught: true,
+                  code_version: "${it.blueOceanPluginVersion}",
+                  source_map_enabled: true,
+                  guess_uncaught_frames: true,
+                  transform: transformer
+                };
+                <![CDATA[
```

Comment:
@kzantow Yes, It doesn't make to browser, wrapping in CDATA is done so that stapler jelly tag <st:include> doesn't get to parse this.